### ISSUE TITLE
Using with lint-staged failed with "'<filename>' is not included in the project."

### ISSUE THREAD

**Issue by gustofbreak** [fimbullinter/wotan#386]
## Meta

wotan 0.12.0 (Running `node_modules/.bin/wotan --version` says "Unknown option '--version'." BTW.)
tsc 3.0.1

How did you run wotan? CLI

## Configuration

```yaml
rules:
  await-async-result:
    rule: await-async-result
    severity: error
  await-only-promise:
    rule: await-only-promise
    severity: error
  ban-dom-globals:
    rule: ban-dom-globals
    severity: error
  delete-only-optional-property:
    rule: delete-only-optional-property
    severity: error
  generator-require-yield:
    rule: generator-require-yield
    severity: error
  new-parens:
    rule: new-parens
    severity: error
  no-case-declaration:
    rule: no-case-declaration
    severity: error
  no-debugger:
    rule: no-debugger
    severity: error
  no-duplicate-case:
    rule: no-duplicate-case
    severity: error
  no-duplicate-spread-property:
    rule: no-duplicate-spread-property
    severity: error
  no-fallthrough:
    rule: no-fallthrough
    severity: error
  no-inferred-empty-object:
    rule: no-inferred-empty-object
    severity: error
  no-invalid-assertion:
    rule: no-invalid-assertion
    severity: error
  no-misused-generics:
    rule: no-misused-generics
    severity: error
  no-nan-compare:
    rule: no-nan-compare
    severity: error
  no-octal-escape:
    rule: no-octal-escape
    severity: error
  no-return-await:
    rule: no-return-await
    severity: error
  no-unassigned-variable:
    rule: no-unassigned-variable
    severity: error
  no-unreachable-code:
    rule: no-unreachable-code
    severity: error
  no-unsafe-finally:
    rule: no-unsafe-finally
    severity: error
  no-unstable-api-use:
    rule: no-unstable-api-use
    severity: error
  no-unused-expression:
    rule: no-unused-expression
    severity: error
  no-unused-label:
    rule: no-unused-label
    severity: error
  no-useless-assertion:
    rule: no-useless-assertion
    severity: 'off'
  no-useless-declare:
    rule: no-useless-declare
    severity: error
  no-useless-initializer:
    rule: no-useless-initializer
    severity: error
  no-useless-jump-label:
    rule: no-useless-jump-label
    severity: error
  no-useless-predicate:
    rule: no-useless-predicate
    severity: 'off'
  no-useless-spread:
    rule: no-useless-spread
    severity: error
  prefer-const:
    rule: prefer-const
    severity: error
  prefer-dot-notation:
    rule: prefer-dot-notation
    severity: error
  prefer-for-of:
    rule: prefer-for-of
    severity: error
  prefer-namespace-keyword:
    rule: prefer-namespace-keyword
    severity: error
  prefer-number-methods:
    rule: prefer-number-methods
    severity: error
  prefer-object-spread:
    rule: prefer-object-spread
    severity: error
  return-never-call:
    rule: return-never-call
    severity: error
  syntaxcheck:
    rule: syntaxcheck
    severity: error
  trailing-newline:
    rule: trailing-newline
    severity: error
  try-catch-return-await:
    rule: try-catch-return-await
    severity: error
  typecheck:
    rule: typecheck
    severity: error
```

## Code

When I use Wotan with lint-staged, no matter how I set `include` in tsconfig.json, it fails with "... is not included in the project." It seems lint-staged set file paths as absolute, but Wotan rejects absolute path when tsconfig.json is specified, which is required for my setup.

I tried this code:

package.json

```json
{
  "some": "settings",
  "scripts": {
    "lint:wotan": "wotan -p ./tsconfig.json",
    "precommit": "lint-staged"
  },
  "other settings": "like dependencies"
}
```

lint-staged.config.js

```js
module.exports = {
  '*.js': 'yarn lint:wotan',
  '*.jsx': 'yarn lint:wotan',
  '*.ts': 'yarn lint:wotan',
  '*.tsx': 'yarn lint:wotan'
};
```

I expected to see this happen: I think when the command is used with specific paths, file inclusion information in tsconfig.js should be ignored, in accordance with tsc. ("When input files are specified on the command line, `tsconfig.json` files are ignored.", <https://www.typescriptlang.org/docs/handbook/tsconfig-json.html>)

Instead, this happened: Testing with various style file paths shows the following results.

```bash
$ wotan -p ./tsconfig.json relative/path/without/dot.ts # success
$ wotan -p ./tsconfig.json ./relative/path/with/dot.ts # fail
$ wotan -p ./tsconfig.json /apsolute/path.ts #fail
```

Related package versions: husky@0.14.3, lint-staged@7.2.2

**Comment by gustofbreak** [comment]
Oh, I forgot to mention #98, which seems to be related.

**Comment by ajafff** [comment]
I'll investigate why absolute paths are not handled correctly.

> Running node_modules/.bin/wotan --version says "Unknown option '--version'." BTW.

Looks like I implemented it as `wotan version`. That's not intentional. Will be fixed in the next release.  Thank you for reporting.

**Comment by ajafff** [comment]
This is indeed related to absolute paths. The current implementation doesn't work very well (if at all) when using absolute paths.
The fix is up at #388

**Comment by ajafff** [comment]
This is fixed in `v0.13.0`
